app/models/metadata_presenter/next_page.rb

```diff
@@ -11,17 +11,53 @@ def find
       else
         service.find_page_by_uuid(current_page_flow.default_next)
       end
+    ensure
+      session[:return_to_check_your_answer] = nil
```

Comment:
👍  nice. was gonna ask about this

Reply:
💯 Although this will require change when we decide to do the magic return to cya in the future far away.